### PR TITLE
Add Order functions and initialization topics

### PR DESCRIPTION
**Pull request by keharper**
This PR adds content for the Order function and initialization topics.

### CODE REVIEW

File: src/content/docs/dropins/order/functions.mdx [hunk 1]
```diff
@@ -8,3 +8,542 @@ tableOfContents:
...
+
+Returns a promise that resolves to a `TransformedData` object.
+
+### Usage
+
+type QueryType = 'orderData';
+
+```ts
+getOrderDetailsById(
+  orderId?: 12345;
```
Comment (KostiantynFandeliuk) [inline src/content/docs/dropins/order/functions.mdx]: Could you change type `orderId?: string;`, please. Thank you.

File: src/content/docs/dropins/order/functions.mdx [hunk 2]
```diff
@@ -8,3 +8,542 @@ tableOfContents:
...
+  }[];
+}
+```
+
+### Usage
+
+The following example demonstrates how to reorder items from a previous order:
+
+```ts
+reorderItems(orderNumber: '12345');
```
Comment (KostiantynFandeliuk) [inline src/content/docs/dropins/order/functions.mdx]: Could you change type orderId?: string;, please. Thank you.
If you wanted to use the example order 000004355

File: src/content/docs/dropins/order/functions.mdx [hunk 3]
```diff
@@ -8,3 +8,542 @@ tableOfContents:
 
 import Aside from '@components/Aside.astro';
 import OptionsTable from '@components/OptionsTable.astro';
+import CodeInclude from '@components/CodeInclude.astro';
+import orderDataModel from '_includes/order-data-model.ts?raw';
+
+The Order Drop-in component provides the following API endpoints, allowing developers to perform various account-related operations and create custom implementations:
```
Comment (bdenham) [inline src/content/docs/dropins/order/functions.mdx]:
```suggestion
The Order drop-in component provides the following API endpoints, allowing developers to perform various account-related operations and create custom implementations:
```

**Review by bdenham — APPROVED** [review]
Just a few changes to make first, then approved.

**Pull request merged**